doc/source/management/data_at_rest_encryption.rst

```diff
@@ -477,202 +597,118 @@ System Variables
 
 The variable turns on binary and relay logs encryption.
 
+.. rubric:: Redo log encryption
 
+:Availability: This feature is **Experimental** quality.
```

Comment:
Master key redo log encryption is GA in 8.0.16, keyring_key is experimental.

Reply:
Fixed